src/core/Akka.Cluster.TestKit/MultiNodeClusterSpec.cs

```diff
@@ -63,7 +68,7 @@ public static Config ClusterConfig()
                 akka.coordinated-shutdown.terminate-actor-system = off
                 #akka.loggers = [""Akka.TestKit.TestEventListener, Akka.TestKit""]
                 akka.test {
-                    single-expect-default = 15 s
```

Comment:
This is causing some of the test failures in sharding.
e.g. some `AwaitAssert` with 10 sec timeouts have no chance to retry when message gets dropped because of rebalance with the timeout of 15 sec.
Question is, what's better. Changing this to 5 sec, same as in scala, maybe together with using `timefactor` > 1.
Or keeping this, and in some tests overriding it or increasing those timeouts

Reply:
for now overwritten only for sharding tests

Reply:
:+1: